src/app/person/person.cpp

```diff
@@ -32,11 +35,78 @@ std::string Person::getSurname() const
 
 void Person::setPesel(const std::string& pesel)
 {
-    // TODO Validation
-    pesel_ = pesel;
+    if (validatePESEL(pesel))
+    {
+        pesel_ = pesel;
+    }
+    // TODO: Else what?
```

Comment:
Instead if condition maybe throw some exception.

Reply:
Condition has been removed, currently there is no pesel checking in function setPesel()